dashboard/test/ui/features/gdpr_dialog.feature

```diff
@@ -13,45 +13,16 @@ Feature: GDPR Dialog - data transfer agreement
     Then I wait to see ".header_user"
     Then element ".ui-test-gdpr-dialog" is not visible
 
-  @skip
-  Scenario: EU user sees the GDPR Dialog on pegasus, opt out
-    Given I am on "http://code.org/"
-    And I am a teacher
-    And I am on "http://code.org/teacher-dashboard?no_home_redirect=1&force_in_eu=1"
-    Then I wait to see ".ui-test-gdpr-dialog"
-    Then I click selector ".ui-test-gdpr-dialog-logout"
-    Then I wait until I am on "http://code.org/"
-    Then I am not signed in
-    Then I wait to see ".header_user"
-    Then element ".ui-test-gdpr-dialog" is not visible
-
```

Comment:
I think removing the Pegasus teacher-dashboard means we don't even need the GDPR Dialog on Pegasus.  Is that true @poorvasingal? 

If so, I can own follow-up (basically reverse of #22595)